com.woltlab.wcf/templates/boxStaffOnline.tpl

```diff
@@ -1,6 +1,8 @@
 <ul class="sidebarItemList">
 	{foreach from=$usersOnlineList item=userOnline}
 		<li class="box32">
+			{event name='staffOnlineListItem'}
```

Comment:
See above, this one conflicts with `box32`.

Reply:
Here it would be nice to have an event between `{foreach}` and `{/foreach}`.